lineapy/plugins/argo_pipeline_writer.py

```diff
@@ -118,6 +117,12 @@ def _write_operators(
         for parameter_name, input_spec in super().get_pipeline_args().items():
             input_parameters_dict[parameter_name] = input_spec.value
 
+        # set kube config to be user given path or expand default ~/.kube/config
+        if "kube_config" in self.dag_config:
+            kube_config = f'"{self.dag_config.get("kube_config")}"'
+        else:
+            kube_config = 'os.path.expanduser("~/.kube/config")'
```

Comment:
adding a previously unrequired dependency on os package. its currently imported in argo jinja template but might need to be careful when doing this for others.

Reply:
👍 I checked for argo beforehand yeah, will need to be careful for other frameworks I totally agree